tslint-server/package.json

```diff
@@ -16,7 +16,7 @@
     "@types/node": "^6.0.41",
     "@types/semver": "^5.3.30",
     "typescript": "^2.0.3",
-    "tslint": "^3.15.1"
+    "tslint": "^4.0.1"
```

Comment:
Did you want to increase the package version for tslint-server and the parent package?